platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/JsonUtilsTest.java

```diff
@@ -82,35 +82,37 @@ public void testConvertToMapWhenNotMap() throws Exception {
   @Test
   public void testGetMapFrom() throws Exception {
     Assert.assertThat(
-        JsonUtils.getMapFrom(JsonUtilsTest.JSON_DEEP_MAP, "map"),
+        JsonUtils.getMapFrom(JsonUtilsTest.JSON_DEEP_MAP, "map", false),
```

Comment:
❗️ I would expect to see tests with `true` as well now that there is a new parameter.